moped-editor/src/views/projects/projectView/ProjectComponents/mapSettings.js

```diff
@@ -69,13 +69,14 @@ export const basemaps = {
     },
     layers: {
       aerials: {
-        id: "simple-tiles",
+        id: "aerial-tiles",
         type: "raster",
-        source: "raster-tiles",
+        source: "aerial-tiles",
         minzoom: 0,
         maxzoom: 22,
       },
       streetLabels: {
+        id: "street-labels",
```

Comment:
Adding this id here so that we can target it with the [`beforeId`](https://visgl.github.io/react-map-gl/docs/api-reference/layer#beforeid) prop of other layers so that these street labels are always on top